jetty-core/jetty-io/pom.xml

```diff
@@ -50,6 +55,22 @@
           <argLine>@{argLine} ${jetty.surefire.argLine} 
             --add-reads org.eclipse.jetty.io=org.eclipse.jetty.logging</argLine>
         </configuration>
+        <dependencies>
+          <!--
```

Comment:
It's ugly, but maybe I could:
- Create pull request to the reactive streams TCK and support also junit platform
- Have a look at the surefire plugin and try to understand if it possible to make junit5 with testng working together out of the box

Reply:
@olamy ???

Reply:
yup no much choice.

Reply:
It would be good to add parsing of `testng` result files for Jenkins.
```
olamy@pop-os:~/dev/sources/jetty/jetty.project$ ls -lrt jetty-core/jetty-io/target/surefire-reports/Surefire\ suite/
total 72
-rw-rw-r-- 1 olamy olamy 60957 May 22 12:43 'Surefire test.html'
-rw-rw-r-- 1 olamy olamy  7951 May 22 12:43 'Surefire test.xml'
-rw-rw-r-- 1 olamy olamy  2127 May 22 12:43  testng-failed.xml

```
See the Jenkinsfille 
transform 
```
junit testResults: '**/target/surefire-reports/*.xml,**/target/invoker-reports/TEST*.xml', allowEmptyResults: true
```
into 
```
junit testResults: '**/target/surefire-reports/**/*.xml,**/target/invoker-reports/TEST*.xml', allowEmptyResults: true
```
This should make it (at least we will see :))

Reply:
@olamy if you have time, could you please update jenkins configuration, because I'm not so much familiar with it

Reply:
I cannot push to your fork/branch. So you will need to cherry pick this commit https://github.com/jetty/jetty.project/commit/5f0168b96a27c852969fc7291bf7f750f543579d

Reply:
Is this new testng requirement because it's used by `import import org.reactivestreams.tck.TestEnvironment;` ?

Reply:
@joakime unfortunately yes https://github.com/reactive-streams/reactive-streams-jvm/tree/master/tck

I'm not a fun to mix multiple testing frameworks, but it is what it is